test-integration/samples/.jhipster/BankAccount.json

```diff
@@ -23,7 +23,7 @@
         { "fieldName": "description", "fieldType": "byte[]", "fieldTypeBlobContent": "text" }
     ],
     "changelogDate": "20200804163352",
-    "dto": "no",
+    "dto": "yes",
```

Comment:
Is this value work? Because in JHipster the other option used is `  "dto": "mapstruct",` (yes I know this is lacking logic).

Reply:
Done. Value changed to mapstruct